site-content/source/modules/ROOT/pages/community.adoc

```diff
@@ -45,70 +45,94 @@ Read the https://www.apache.org/foundation/policies/conduct[Apache Code of Condu
 == Discussions
 If you are a Cassandra user or want to learn more, we invite you to join these channels created specifically for end user questions and discussions.
 
-// start discussion row
+// start discussion row     >>>>>>>>>> Q&A Forum
 [openblock, discussion-row text-left cf]
 --------
 
 [openblock, float33 full-800]
 ----------
 [discrete]
-=== User Mailing List
+=== Q&A Forums
 ----------
 
 [openblock, float66 full-800]
 ----------
-General mailing list for user questions and discussions. This is also where new releases are announced.
+For questions about developing apps or need help operating Cassandra, post on the Stack Exchange network.
```

Comment:
Stack network or we need to move both sites

Reply:
Updated to `Stack network`. 👍